webrender_traits/src/channel.rs

```diff
@@ -2,10 +2,52 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
+use byteorder::{LittleEndian, WriteBytesExt, ReadBytesExt};
+use std::io::{Cursor, Read};
+use api::Epoch;
+
+#[derive(Clone)]
+pub struct Payload {
+    pub epoch: Epoch,
+    pub display_list_data: Vec<u8>,
+    pub auxiliary_lists_data: Vec<u8>
+}
+
+impl Payload {
+    pub fn to_data(&self) -> Vec<u8> {
```

Comment:
nit: consider using `From`/`Into` traits. Not really helping anything, just seems to be more idiomatic